.github/workflows/publish-to-npm.yml

```diff
@@ -67,6 +67,9 @@ jobs:
             - name: Install dependencies
               run: yarn
 
+            - name: Build packages
```

Comment:
i think the copy script that bumps the versions should go first, it will modify the src/package.json files, if you do the build first, you end up with old dist/package.json